extensions/wikia/TemplateClassification/specials/TemplatesSpecialController.class.php

```diff
@@ -340,6 +340,6 @@ private function getDefaultType() {
 	 * @return bool
 	 */
 	private function isUserType( $type ) {
```

Comment:
Can we move this method into UserTemplateClassificationService and make UserTemplateClassificationService::$templateTypes private in return?